src/notification/slack-notify.ts

```diff
@@ -9,52 +9,44 @@ import { NotificationComment, SlackAttachment, SlackMessage } from './types';
 export function getMessage(
```

Comment:
It seems this function is not used anywhere else, so I'd drop the `export` to make that explicit.